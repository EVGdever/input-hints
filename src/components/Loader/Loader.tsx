import React from "react";
import loader from "../../images/loader.png";

const Loader: React.FC = () => {
  return (
    <div className="loader" >
      <img src={loader} alt="loader"/>
    </div>
  )
}

export default Loader;