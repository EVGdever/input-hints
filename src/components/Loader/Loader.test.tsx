import React from 'react';
import {shallow} from 'enzyme';
import Loader from "./Loader";

describe('Loader', () => {
  let wrapper;

  it('should render a loader', () => {
    wrapper = shallow(<Loader/>)
    expect(wrapper).toMatchSnapshot();
  });
})