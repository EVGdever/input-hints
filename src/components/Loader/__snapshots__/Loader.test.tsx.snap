// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should render a loader 1`] = `
<div
  className="loader"
>
  <img
    alt="loader"
    src="image/mock/url.png"
  />
</div>
`;
